src/mlpack/methods/ann/activation_functions/rectifier_function.hpp

```diff
@@ -24,6 +24,7 @@
 #define MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_RECTIFIER_FUNCTION_HPP
 
 #include <mlpack/prereqs.hpp>
+#include <mlpack/core/math/mat_redef.hpp>
```

Comment:
I think it would be fine to include this as part of `prereqs.hpp` (or wherever it is that we include Armadillo).